src/foundry/foundry.js/clientDocuments/user.d.ts

```diff
@@ -10,7 +13,135 @@ declare global {
    *
    * @param data - Initial data provided to construct the User document
    */
-  class User extends ClientDocumentMixin(foundry.documents.BaseUser) {}
+  class User extends ClientDocumentMixin(foundry.documents.BaseUser) {
+    /**
+     * Track whether the user is currently active in the game
+     */
+    active: boolean;
+
+    /**
+     * Track references to the current set of Tokens which are targeted by the User
+     */
+    targets: UserTargets;
+
+    /**
+     * Track the ID of the Scene that is currently being viewed by the User
+     */
+    viewedScene: string | null;
+
+    /**
+     * Return the User avatar icon or the controlled actor's image
+     */
+    get avatar(): string;
+
+    /**
+     * Return the Actor instance of the user's impersonated character (or undefined)
+     */
+    get character(): ConfiguredDocumentClass<typeof Actor> | undefined;
```

Comment:
Might make sense to reference the return type via `ReturnType<Game['actors']['get']>` If that is already possible.